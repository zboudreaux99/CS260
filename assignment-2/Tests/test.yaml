test:
  timelimit: 10
  launch: ./tally
  params: '@T/albums.csv @T/requests.csv'
  stderr: false
dueDate: 12/06/2024
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["playlist.h", "playlist.cpp"]
  instructorFiles:
    exclude: ["playlist.h", "playlist.cpp"]
