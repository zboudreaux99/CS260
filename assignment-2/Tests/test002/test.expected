Read 83 songs.
Processed 3 playlist items.

New Song                 The Who             Who Are You              1 04:17
Had Enough               The Who             Who Are You              2 04:30
905                      The Who             Who Are You              3 03:56
Sister Disco             The Who             Who Are You              4 04:24
Music Must Change        The Who             Who Are You              5 04:40
Trick of the Light       The Who             Who Are You              6 04:14
Guitar and Pen           The Who             Who Are You              7 05:50
Love is Coming Down      The Who             Who Are You              8 04:02
Who Are You              The Who             Who Are You              9 06:24
Sgt. Peppers Lonely Hear The Beatles         Sgt. Peppers Lonely Hear 1 02:10
A Little Help From My Fr The Beatles         Sgt. Peppers Lonely Hear 2 02:36
Lucy In The Sky With Dia The Beatles         Sgt. Peppers Lonely Hear 3 03:26
Getting Better           The Beatles         Sgt. Peppers Lonely Hear 4 02:46
Fixing A Hole            The Beatles         Sgt. Peppers Lonely Hear 5 02:35
She's Leaving Home       The Beatles         Sgt. Peppers Lonely Hear 6 03:33
Being For The Benefit Of The Beatles         Sgt. Peppers Lonely Hear 7 02:34
Within You Without You   The Beatles         Sgt. Peppers Lonely Hear 8 05:03
When I'm Sixty-Four      The Beatles         Sgt. Peppers Lonely Hear 9 02:38
Lovely Rita              The Beatles         Sgt. Peppers Lonely Hear 10 02:42
Good Morning, Good Morni The Beatles         Sgt. Peppers Lonely Hear 11 02:41
Sgt. Peppers Lonely Hear The Beatles         Sgt. Peppers Lonely Hear 12 01:22
A Day In The Life        The Beatles         Sgt. Peppers Lonely Hear 13 04:58
End                      The Beatles         Sgt. Peppers Lonely Hear 14 00:35
Beastie                  Jethro Tull         The Broadsword and the B 1 03:58
Clasp                    Jethro Tull         The Broadsword and the B 2 04:17
Fallen on Hard Times     Jethro Tull         The Broadsword and the B 3 03:13
Flying Colours           Jethro Tull         The Broadsword and the B 4 04:39
Slow Marching Band       Jethro Tull         The Broadsword and the B 5 03:42
Broadsword               Jethro Tull         The Broadsword and the B 6 05:03
Pussy Willow             Jethro Tull         The Broadsword and the B 7 03:55
Watching Me Watching You Jethro Tull         The Broadsword and the B 8 03:41
Seal Driver              Jethro Tull         The Broadsword and the B 9 05:10
Cheerio                  Jethro Tull         The Broadsword and the B 1 01:12
Living In The Past       Jethro Tull         The very best of Jethro  1 03:23
Aqualung                 Jethro Tull         The very best of Jethro  2 06:39
Sweet Dream              Jethro Tull         The very best of Jethro  3 04:06
The Whistler             Jethro Tull         The very best of Jethro  4 03:33
Bungle In The Jungle     Jethro Tull         The very best of Jethro  5 03:39
The Witch's Promise      Jethro Tull         The very best of Jethro  6 03:53
Locomotive Breath        Jethro Tull         The very best of Jethro  7 04:27
Steel Monkey             Jethro Tull         The very best of Jethro  8 03:39
Thick As A Brick         Jethro Tull         The very best of Jethro  9 03:05
Bouree                   Jethro Tull         The very best of Jethro  10 03:48
Too Old To Rock'N'Roll:  Jethro Tull         The very best of Jethro  11 03:57
Life Is A Long Song      Jethro Tull         The very best of Jethro  12 03:20
Songs From The Wood      Jethro Tull         The very best of Jethro  13 04:55
A New Day Yesterday      Jethro Tull         The very best of Jethro  14 04:11
Heavy Horses             Jethro Tull         The very best of Jethro  15 03:22
Broadsword               Jethro Tull         The very best of Jethro  16 05:02
Roots To Branches        Jethro Tull         The very best of Jethro  17 05:13
A Song For Jeffrey       Jethro Tull         The very best of Jethro  18 03:20
Total: 03:14:18

